browser-test/src/support/admin_predicates.ts

```diff
@@ -48,7 +44,8 @@ export class AdminPredicates {
     }
 
     await this.clickAddConditionButton()
-    await this.addValueRows(predicateSpecs[0])
+    const totalRowsNeeded = predicateSpecs[0]?.values?.length ?? 0
+    await this.addValueRows(Math.max(totalRowsNeeded - 1, 0))
```

Comment:
(question) Is this adding one less rows because there's already one on the screen by default?

Reply:
Yup - the existing code was doing that already, it was just less obvious because the for loop started at 1 instead of 0